src/clr-angular/forms/datepicker/date-container.ts

```diff
@@ -33,7 +33,14 @@ import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
         <div class="clr-input-wrapper">
           <div class="clr-input-group" [class.clr-focus]="focus">
             <ng-content select="[clrDate]"></ng-content>
-            <button #actionButton type="button" class="clr-input-group-icon-action" (click)="toggleDatepicker($event)" *ngIf="isEnabled" [attr.title]="commonStrings.open" [disabled]="control?.disabled">
+            <button #actionButton 
+                    type="button"
+                    class="clr-input-group-icon-action"
+                    [attr.title]="commonStrings.openDatepicker"
+                    [attr.aria-label]="commonStrings.openDatepicker"
```

Comment:
Why are we setting both `aria.title` and `attr.aria-label`? Are they not the same when it comes to screen reader?

Reply:
Title is needed for tooltips when mouse hovers the button. aria-label is needed to ensure screen readers (that don't read title) will read it.  

@chlane - did I miss anything?